akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -163,26 +164,43 @@ abstract class ActorMaterializer extends Materializer {
   def isShutdown: Boolean
 
   /**
-   * INTERNAL API: this might become public later
+   * INTERNAL API
    */
-  private[akka] def actorOf(context: MaterializationContext, props: Props): ActorRef
+  def actorOf(context: MaterializationContext, props: Props): ActorRef
```

Comment:
can this still be `private[akka]` here and public in `ExtendedActorMaterializer`?